Commit_1

Gomina opened the pull request:
Созданы тесты и дополнительные файлы для их выполнения.

Gomina committed:
Commit_1
Созданы тесты и дополнительные файлы для их выполнения.

DolgoarshinnykhLG commented on pages/home_page.py:
```diff
@@ -0,0 +1,68 @@
+        # Создаем экземпляр WebDriverWait с таймаутом 10 секунд
+        self.wait = WebDriverWait(self.driver, 10)
+
+    @allure.title('метод открывает сайт на главной странице')
```
Необходимо исправить здесь и далее: allure.title следует использовать для аннотаций тестов. Для шага теста (или метода вызываемого в нём) лучше подходит allure.step

DolgoarshinnykhLG commented on test/test_home_page_questions_about_important.py:
```diff
@@ -0,0 +1,43 @@
+        # проскролить до элемента и кликнуть его
+        page.scroll_to_element(locator_question)
+        # дождаться пока текст появится на странице, проверить на соответствие ОР и ФР
+        WebDriverWait(page.driver, 10).until(
```
Нужно исправить: здесь и далее, все вызовы к driver/WebDriverWait напрямую из теста и/или фикстуры не допустимы (кроме get). Необходимо реализовать соответствующий метод в *Page и вызывать его

DolgoarshinnykhLG commented on test/test_order_scooter.py:
```diff
@@ -0,0 +1,121 @@
+        page.clic_on_element(TLOS.LOCATOR_SCOOTER_LOGO)
+        # дождаться загрузки домашней страницы
+        WebDriverWait(driver, 5).until(
+            EC.url_to_be("https://qa-scooter.praktikum-services.ru/")
```
Можно лучше: url-адреса лучше хранить отдельно, они постоянно переиспользуются. Можно вынести в модуль urls

DolgoarshinnykhLG commented on test/test_home_page_questions_about_important.py:
```diff
@@ -0,0 +1,43 @@
+        ]
+    )
+
+    def test_questions_about_important(self, driver, locator_question, locator_answer, answer_text):
```
Необходимо исправить: над каждым тестом должна быть соответствующая аннотация allure.title

Gomina committed:
Commit_2
Исправлены allure.title на allure.step

Gomina committed:
Commit_3
Над всеми тестами добавлены аннотации allure.title

Gomina committed:
Commit_4
Из тестов убран driver/WebDriverWai. Создана функция в Page
URL берется из файла файла urls.py

Gomina committed:
Commit_5
После внесенных изменений сгенерирован новый отчет
